src/Employer/Employer.Web/Mappings/TrainingProviderMapper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using SFA.DAS.Apprenticeships.Api.Types.Providers;
+using System.Linq;
+
+namespace Esfa.Recruit.Employer.Web.Mappings
+{
+    public static class TrainingProviderMapper
+    {
+        public static TrainingProvider MapFromApiProvider(Provider provider)
+        {
+            return new TrainingProvider
+            {
+                Ukprn = provider.Ukprn,
+                Name = provider.ProviderName,
+                Address = GetAddress(provider.Addresses.FirstOrDefault(addr => addr.ContactType.Equals("PRIMARY")) ?? provider.Addresses.First())
+            };
+        }
+        
+        private static Address GetAddress(ContactAddress address)
+        {
+            return new Address
+            {
+                AddressLine1 = address.Primary,
+                AddressLine2 = address.Secondary,
```

Comment:
If address.Secondary is empty should we put in address.Street instead. And so on....

Reply:
This is sorted on the provider API (I hope, I was the one that did the work 😬 ) when it gets its data from UKRLP. Will double check!

Reply:
It should be okay since we are reducing when displaying the address.